### PR TITLE
(DOCSP-1134): Added details to verbose logging option.

### PR DESCRIPTION
**Pull request by atsansone**
@i80and : Could you review because Jeff Allen does not have access?

### CODE REVIEW

File: source/includes/options-mongosqld.yaml [hunk 1]
```diff
@@ -32,8 +32,32 @@ name: verbose
 directive: option
 aliases: -v
 description: |
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  Can be specified multiple times or with an integer to provide greater
```
Comment (i80and) [inline source/includes/options-mongosqld.yaml]: `This option can...`

File: source/includes/options-mongosqld.yaml [hunk 2]
```diff
@@ -32,8 +32,32 @@ name: verbose
...
+  application detail.
```
Comment (i80and) [inline source/includes/options-mongosqld.yaml]: Maybe add `The following table describes each of the available log levels`?

File: source/includes/options-mongosqld.yaml (hunk 1 above)
Comment (i80and) [reply source/includes/options-mongosqld.yaml]: Actually, I would prefer "You can provide this flag repeatedly or with an integer...`

File: source/includes/options-mongosqld.yaml [hunk 3]
```diff
@@ -32,8 +32,34 @@ name: verbose
 directive: option
 aliases: -v
 description: |
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  You can provide this option repeatedly (up to three times) or with an
+  integer between ``1`` and ``3`` to provide greater application detail.
+  The following tale describes the information provided at each log
```
Comment (i80and) [inline source/includes/options-mongosqld.yaml]: I think you meant `table` ;)

**Comment by atsansone** [comment]
@rychipman : Please have a look. I believe this covers the logging verbosity. I know that the logging levels are being investigated currently, so if this needs an update, cool.

File: source/includes/options-mongosqld.yaml [hunk 4]
```diff
@@ -32,8 +32,34 @@ name: verbose
...
-  Specifies that {{program}} should provide more detailed log output. Include
-  multiple times for more verbosity (e.g. ``-vvvvv``).
+  Specifies that {{program}} should provide more detailed log output.
+
+  You can provide this option repeatedly (up to three times) or with an
+  integer between ``1`` and ``3`` to provide greater application detail.
+  The following table describes the information provided at each log
+  level:
+
+  .. list-table::
```
Comment (deafgoat) [inline source/includes/options-mongosqld.yaml]: here's what we're currently working towards: https://docs.google.com/a/10gen.com/document/d/1VEzEejIrOELcHAZa2XSmiawvwTFHM0g9X5bM43OLB60/edit?usp=sharing (we also have `--quiet`)

a more detailed description of what to expect at each verbosity level is provided there

**Comment by atsansone** [comment]
@deafgoat : I have updated per the [Google Doc](https://docs.google.com/document/d/1VEzEejIrOELcHAZa2XSmiawvwTFHM0g9X5bM43OLB60/edit) and BI-1001 having been completed. Can you review and provide revisions or approval?

File: source/includes/options-mongosqld.yaml [hunk 5]
```diff
@@ -32,8 +32,34 @@ name: verbose
...
+     * - No flag
+       - Default
+       - Log messages that notify user of basic :program:`mongosqld` events and state changes.
+     * - ``-v``
+       - Administrator
+       - Log messages that provie information to :program:`mongosqld` administrators.
+     * - ``-vv``
+       - Developer
+       - Log messages useful to MongoDB support and development.
+     * - ``-vvv``
```
Comment (rychipman) [inline source/includes/options-mongosqld.yaml]: Trace was removed in the last iteration of the code review, but that change had apparently never made it back to the doc. I've updated the doc accordingly.

This means we can just remove this bullet point

File: source/reference/log-messages.txt [hunk 6]
```diff
@@ -92,26 +92,24 @@ categorization of the messages:
...
-to ``3``.
+To specify a verbosity level when starting ``mongosqld``, use the
+``-v`` option. Repeating ``v`` increases the verbosity level.
 
-.. code-block:: shell
+.. example::
 
-   mongosqld --schema schema.drdl --auth -vvv
+   The following invocation of ``mongosqld`` sets the log verbosity
+   level to Trace.
```
Comment (rychipman) [inline source/reference/log-messages.txt]: we should change this example to use a level that isn't Trace

**Comment by deafgoat** [comment]
Nothing to add besides Ryan's recent comments.

**Pull request merged**